demo/index.html

```diff
@@ -146,25 +146,30 @@
 
           videos: {
             type: Array,
-            value: []
+            value: function() {
+              return [];
+            }
           },
 
           data: {
             type: Object,
-            value: {
-              page: '/search/'
+            value: function() {
+              return {
+                page: '/search/'
+              };
             }
-          }
+          },
```

Comment:
IE 9 and above behave totally consistently with trailing commas! A semicolon for every statement and a trailing comma for every literal entry!